buffalo/cmd/build.go

```diff
@@ -343,7 +346,24 @@ func (b *builder) buildBin() error {
 			version = strings.TrimSpace(out.String())
 		}
 	}
-	buildArgs = append(buildArgs, "-ldflags", fmt.Sprintf("-X main.version=%s -X main.buildTime=%s", version, buildTime))
+
+	flags := []string{
+		fmt.Sprintf("-X main.version=%s", version),
+		fmt.Sprintf("-X main.buildTime=%s", buildTime),
+	}
+
+	// Add any additional ldflags passed in to the build args
+	if len(ldflags) > 0 {
+		if foundVersion, _ := regexp.MatchString("-X\\s+main.version=", ldflags); foundVersion {
+			return errors.New("The ldflag option '-X main.version=' is reserved for gobuffalo use")
```

Comment:
Go prefers errors to start with a lower case. Also please change `gobuffalo` to `Buffalo`.